src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days > 6) {
+    return days * 40 - 50;
+  } else if (days > 2) {
+    return days * 40 - 20;
+  } else {
+    return days * 40;
```

Comment:
create vars for the price and discounts
it is easier to change values than to search for numbers